### PR TITLE
Genetic Sort

### PR DESCRIPTION
**Pull request by clamli**
Add data/path function for Genetic Sort.

Boyi

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,21 +3,22 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "NODE_ENV=dev node app.js",
-    "prod": "NODE_ENV=prod node app.js",
+    "start": "cross-env NODE_ENV=dev node app.js",
+    "prod": "cross-env NODE_ENV=prod node app.js",
```
Comment (ishuen) [inline package.json]: We can use different command name. Maybe change start to start-u for unix and start-w for windows.

File: views/geneticSort.pug [hunk 2]
```diff
@@ -0,0 +1,95 @@
+doctype html
+html
+  head
+    meta(charset='UTF-8')
+    title Topological Sorting
+  style.
+    .axis path,
```
Comment (ishuen) [inline views/geneticSort.pug]: Better write this part in public/css. It's okay to remain this part unchanged. If you already run out of the time, I will move it by myself.

File: npm-debug.log [hunk 3]
```diff
@@ -0,0 +1,112 @@
+0 info it worked if it ends with ok
```
Comment (ishuen) [inline npm-debug.log]: This file shouldn't be uploaded. Please add the file name in .gitignore.

File: package.json [hunk 4]
```diff
@@ -3,21 +3,22 @@
...
     "standard": "standard --verbose | snazzy",
     "precommit": "npm run standard"
   },
   "dependencies": {
     "body-parser": "~1.17.1",
```
Comment (ishuen) [inline package.json]: My bad.. Thanks for reordering it by alphabet.

**Comment by ishuen** [comment]
Please upload it after "rebase" to prevent conflict.

File: package.json (hunk 1 above)
Comment (clamli) [reply package.json]: It seems that start-w can't work on my Win.

Comment (ishuen) [reply package.json]: As long as you change the words in the quote mark, you can try every word you want. It should work, e.g. "start-w": "NODE_ENV=dev node app.js". (Though I'm not sure about how they detect the special symbol in win)

File: views/geneticSort.pug (hunk 2 above)
Comment (clamli) [reply views/geneticSort.pug]: done.

File: npm-debug.log (hunk 3 above)
Comment (clamli) [reply npm-debug.log]: done.

**Pull request merged**